WordPress/src/main/res/values/strings.xml

```diff
@@ -886,6 +885,8 @@
     <string name="stats_item_collapsed">Item collapsed</string>
     <string name="stats_posting_activity_graph">Graph of posting activity.</string>
     <string name="stats_list_item_description">%1$s: %2$s, %3$s: %4$s</string>
+    <string name="stats_bar_chart_accessibility_entry">%1$s, %2$d %3$s</string>
+    <string name="stats_bar_chart_accessibility_overlapping_entry">&#160; &amp; %1$d %2$s</string>
```

Comment:
I'm not familiar with the code, could you please help me understand what is an "overlapping entry"? Can I somehow test this behavior?

Reply:
An overlapping entry is when two sets of stats are being represented on a single chart. So when you are able to see both "Views and Visitors" at the same time one of those values is overlapping the other. In this case, it would be the visitors that are overlapping entries. Let me know if this explanation helps!